mockacino.rb

```diff
@@ -95,6 +100,12 @@ def self.listen_address
 post '/:token*' do
   path = params[:token]
   extension = params[:splat].first
+  
+  filePath = File.join(MockacinoConfig.site_root, MockacinoConfig.post_path, path, extension, MockacinoConfig.config_file)
+  if File.exist?(filePath)
+      file = File.read(filePath)
+      status file
```

Comment:
Sounds great.
Now it is key value based, so example config.conf will be :
status, 204